lib/cretonne/src/isa/intel/mod.rs

```diff
@@ -111,4 +117,149 @@ impl TargetIsa for Isa {
     fn reloc_names(&self) -> &'static [&'static str] {
         &binemit::RELOC_NAMES
     }
+
+    fn prologue_epilogue(&self, func: &mut ir::Function) -> result::CtonResult {
+        let word_size = if self.flags().is_64bit() { 8 } else { 4 };
+        let csr_type = if self.flags().is_64bit() {
+            ir::types::I64
+        } else {
+            ir::types::I32
+        };
+        let csrs = abi::callee_saved_registers(&self.shared_flags);
+
+        let mut csr_stack_size = word_size; // Size of RBP to start with
+        for _reg in &csrs {
+            csr_stack_size += word_size;
+        }
```

Comment:
`(csrs.len() + 1) * word_size` ;-)

You could consider including the frame pointer in the set returned by `callee_saved_registers`. We don't support frame pointer elimination at the moment, but when we do, `%rbp` will behave exactly like any other CSR.

Reply:
oh jeez, this one is embarrassing